presto-product-tests/conf/docker/singlenode-ldap-bind-dn/docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: '2'
+services:
+
+  presto-master:
+    image: 'prestodev/centos6-oj8-openldap:${DOCKER_IMAGES_VERSION}'
+    command: /docker/volumes/conf/docker/files/presto-launcher-wrapper.sh singlenode-ldap run
+    extra_hosts:
+       - "${LDAP_SERVER_HOST}:${LDAP_SERVER_IP}"
```

Comment:
I think `LDAP_SERVER_HOST` and `LDAP_SERVER_IP` are redundant.

Let me try to remove them https://github.com/prestosql/presto/pull/1924

Reply:
https://github.com/prestosql/presto/pull/1924 is merged now

Reply:
removed the variables here